lib/compilers/raku.ts

```diff
@@ -0,0 +1,61 @@
+import type {AsmResultSource, ParsedAsmResultLine} from '../../types/asmresult/asmresult.interfaces.js';
```

Comment:
Can you please add the copyright header here (all the other files have a copy, just make sure you copy it and are happy with it, and the year is 2025 :))